tests/lms/config/resources_test.py

```diff
@@ -378,6 +378,33 @@ def test_rpc_server_config(self, lti_launch):
             "allowedOrigins": ["http://localhost:5000"]
         }
 
+    def test_provisioning_enabled_returns_True_if_oauth_consumer_key_in_auto_provisioning(
```

Comment:
Can you help me understand what it means for an `oauth_consumer_key` to be "in" `auto_provisioning` in this test name? As I understand it, going forward, we're looking at the context which is looking at the DB record, right? The naming is throwing me here (and on the next test, too).

Reply:
In the `settings["auto_provisioning"]` which is a list of consumer keys from an envvar. But this whole thing is going away in the next PR anyway